mitty/cli.py

```diff
@@ -193,6 +193,26 @@ def print_variant_model_list(ctx, param, value):
   ctx.exit()
 
 
+@cli.command('sample-genome', short_help='Sample variants to create an individual')
+@click.option('--vcf', type=click.Path(exists=True))
+@click.option('--output-name', type=str)
+@click.option('--sample-name', type=str)
+@click.option('--default-allele-freq', default=0.01, type=float)
+def simulate_variants(graph_genome, output_name, sample_name, default_allele_freq):
```

Comment:
@ozemsbg I'm just curious about the name graph_genome. Why is this used rather than input_vcf or something. Thanks!